### PR TITLE
Move to GitHub actions

### PR DESCRIPTION
**Pull request by oscarbc96**
Replace Travis with GitHub Actions

Adds two pipelines:
- Lint & Test: Runs always executing linting and test on python 3.7 and 3.8
- PyPI release: Runs when tag published

Builds: 
https://github.com/Skyscanner/cfripper/actions/runs/378805525
https://github.com/Skyscanner/cfripper/actions/runs/378849178

### CODE REVIEW

**Review by jsoucheiron — APPROVED** [review]
Nice!

File: .github/workflows/pypi-release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - run: make install
+
+      - name: Build a binary wheel
+        run: python setup.py sdist bdist_wheel
+
+      - name: Publish distribution 📦 to Test PyPI
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          password: ${{ secrets.test_pypi_password }}
+          repository_url: https://test.pypi.org/legacy/
```
Comment (ocrawford555) [inline .github/workflows/pypi-release.yml:30]: Assuming the test push to Test PyPi was a success, is there any need for this section in the YML now? Or can we condition it on certain pre-release tags or something fancy? 😃

Comment (oscarbc96) [reply .github/workflows/pypi-release.yml:30]: Yes, I thought the same, but we haven't done this before. So I thought to deploy to test pypi the same way to normal pypi till we agree how to use that environment.

Comment (ocrawford555) [reply .github/workflows/pypi-release.yml:30]: Fair enough! Approving ..

**Review by ocrawford555 — APPROVED** [review]

**Pull request merged**